opennms-config/src/main/java/org/opennms/netmgt/config/DatabaseSchemaConfigFactory.java

```diff
@@ -389,19 +389,39 @@ public String constructJoinExprForTables(final List<Table> tables) {
      */
     public String addColumn(final List<Table> tables, final String column) throws FilterParseException {
         getReadLock().lock();
-        try {
-            final Table table = findTableByVisibleColumn(column);
-            if(table == null) {
-                final String message = "Could not find the column '" + column +"' in filter rule";
-                throw new FilterParseException(message);
+        if (doesColumnExist(column)) {
```

Comment:
we are bypassing throwing exception and returning  empty. Did you ensure this will not cause any issues in other parts of the code ?